test/README.md

```diff
@@ -0,0 +1,95 @@
+## How To Test Verdaccio
+
+Welcome to the testing folder at Verdaccio. This document aims to help you to go throug how Verdaccio show be tested.
+
+First of all we must explain the stack, we use `jest` and other tools as `supertest` to be able test the API and `puppetter` for End to End testing.
```

Comment:
as => such as
to be able test => to be able to test
puppetter => puppeteer
End to End => End-to-End